pkg/test/csrctrl/controllers/start_csrctrl.go

```diff
@@ -51,27 +52,17 @@ type SignerRootCert struct {
 	Rootcert string
 }
 
-func RunCSRController(signerNames string, appendRootCert bool, config *rest.Config, c <-chan struct{},
-	certChan chan *SignerRootCert,
-) {
+func RunCSRController(signerNames string, appendRootCert bool, c <-chan struct{},
+	clusters cluster.Clusters) []*SignerRootCert {
 	// Config Istio log
 	if err := log.Configure(loggingOptions); err != nil {
 		log.Infof("Unable to configure Istio log error: %v", err)
 		os.Exit(-1)
 	}
-	mgr, err := ctrl.NewManager(config, ctrl.Options{
-		Scheme: scheme,
-		// disable the metric server to avoid the port conflicting
-		MetricsBindAddress: "0",
-	})
-	if err != nil {
-		log.Infof("Unable to start manager error: %v", err)
-		os.Exit(-1)
-	}
-
-	arrSingers := strings.Split(signerNames, ",")
-	signersMap := make(map[string]*signer.Signer, len(arrSingers))
-	for _, signerName := range arrSingers {
+	arrSigners := strings.Split(signerNames, ",")
+	signersMap := make(map[string]*signer.Signer, len(arrSigners))
+	var rootCertSignerArr []*SignerRootCert
```

Comment:
nit: 
```suggestion
	var rootCertSignerArr []SignerRootCert
```

No need to use pointer here since its not mutable